mlflow/tracking/fluent.py

```diff
@@ -212,6 +212,40 @@ def log_metric(key, value):
     MlflowClient().log_metric(run_id, key, value, int(time.time()))
 
 
+def log_metrics(metrics):
+    """
+    Log multiple metrics for the current run, starting a run if no runs are active.
+    :param metrics: Dictionary of metric_name: String -> value: Float
+    :returns: None
+    """
+    run_id = _get_or_start_run().info.run_uuid
+    timestamp = int(time.time())
+    metrics_arr = [Metric(key, value, timestamp) for key, value in metrics.items()]
```

Comment:
Looks like we don't perform the same metric value type validation that we do for `log_metric()`. As a result, the following code successfully writes a metric with the value `"dog"` to the file store. However, this corrupts the file store's run representation, and all subsequent calls to `get_run()` with the specified `run_id` fail.

```
with mlflow.start_run():
    run_id = mlflow.active_run().info.run_uuid
    # Writes the metric string value and fails
    mlflow.log_metrics(metrics={"cat": "dog"})
```

```
from mlflow.tracking import MlflowClient
client = MlflowClient()
# Fails because the run has been corrupted by a metric value of type string.
client.get_run(run_id)
```

Reply:
I still think that it's ideal to perform this kind of validation in the store itself so that every client invocation doesn't have to repeat the same validation logic @mparkhe

Reply:
Ah good catch